pugdebug/gui/main_window.py

```diff
@@ -63,7 +76,8 @@ def setup_settings_window(self):
         self.central_widget_layout.addWidget(self.settings_window, 1, 0, 1, 2)
 
     def setup_toolbar(self):
-        toolbar = QToolBar()
+        toolbar = QToolBar("Main Toolbar")
+        toolbar.setObjectName("main-toolbar")
```

Comment:
Required for saveState() on all QToolbars and QDockWidgets attached to the window.
see http://doc.qt.io/qt-5/qmainwindow.html#saveState